gallery/src/pages/AccessibilityChecklist.tsx

```diff
@@ -0,0 +1,1516 @@
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Have we provided captions and descriptions for prerecorded audio and videos?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external href="https://www.w3.org/WAI/WCAG21/quickref/#captions-prerecorded">
+                Level A
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+          <NxList.Item className={`level-aa ${isAA && 'hidden'}`}>
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Are live captions available for live audio and video recordings?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external href="https://www.w3.org/WAI/WCAG21/quickref/#captions-live">
+                Level AA
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+          <NxList.Item className={`level-aa ${isAA && 'hidden'}`}>
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Have we provided audio description which describes in real-time the contents of all video recordings?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external href="https://www.w3.org/WAI/WCAG21/quickref/#audio-description-prerecorded">
+                Level AA
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+          <NxList.Item className={`level-aaa ${isAAA && 'hidden'}`}>
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Do we have a video recording of an ASL interpreter translating our pre recorded media?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external href="https://www.w3.org/WAI/WCAG21/quickref/#sign-language-prerecorded">
+                Level AAA
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+          <NxList.Item className={`level-aaa ${isAAA && 'hidden'}`}>
+            <NxList.Text>
+              <NxStatefulCheckbox defaultChecked={false}>
+                Have we provided an audio description of the prerecorded media that also includes implicit context that
+                may not be understood through a language translation alone?
+              </NxStatefulCheckbox>
+            </NxList.Text>
+            <NxList.Actions>
+              <NxTextLink external
+                          href="https://www.w3.org/WAI/WCAG21/quickref/#extended-audio-description-prerecorded">
+                Level AAA
+              </NxTextLink>
+            </NxList.Actions>
+          </NxList.Item>
+          <NxList.Item className={`level-aaa ${isAAA && 'hidden'}`}>
```

Comment:
A helper component could take out a lot of the repetition in these list items.  I'd imagine it would take 4 props: the level, whether it's hidden, the fragment identifier part of the link, and the question text